app/views/admin/roles/_appointments_tab.html.erb

```diff
@@ -0,0 +1,11 @@
+<%= render "govuk_publishing_components/components/button", {
+  text: "New appointment",
+  href: new_admin_role_role_appointment_path(role, make_current: true)
+} %>
+
+<%= render partial: "admin/role_appointments/list", locals: {role_appointments: role.role_appointments} %>
```

Comment:
this needs to be rebuilt using the govuk design system HTML classes?

Reply:
Thanks @DilwoarH, should I do this within the scope of this ticket? Or does it fall within the scope of 'role_appointments'?